docs/ProductSpecifications.md

```diff
@@ -1,68 +1,63 @@
-# Product Specifications
+📢 Use this project, [contribute](https://github.com/vtex-apps/store-components) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion). 
 
-## Description
+# Product Specifications
 
 `ProductSpecifications` is a VTEX component that shows the technical specifications of a product.
 This component is meant to be used inside the `store.product` block and consume data from the `product-context`.
 
-:loudspeaker: **Disclaimer:** Don't fork this project, use, contribute, or open issue with your feature request.
-
-## Table of Contents
-
-- [Usage](#usage)
-  - [Blocks API](#blocks-api)
-    - [Configuration](#configuration)
-  - [Styles API](#styles-api)
-    - [CSS Namespaces](#css-namespaces)
+![Screen Shot 2019-12-27 at 14 07 30](https://user-images.githubusercontent.com/27777263/71525823-4bd8a380-28b2-11ea-8d5c-7678426ec1ab.png)
 
-## Usage
+## Configuration
 
-You should follow the usage instruction in the main [README](/README.md#usage).
+1. Import the `vtex.store-component` app to your theme's dependencies in the `manifest.json`;
 
-Just add this component to your `store.product` block:
-
-```js
-"store.product": {
-  "children": ["product-specifications"]
-}
+```json
+  "dependencies": {
+    "vtex.store-components": "3.x"
+  }
 ```
 
-### Blocks API
-
-When implementing this component as a block, various inner blocks may be available. The following interface lists the available blocks within `ProductSpecifications` and describes if they are required or optional.
-
-```js
- "product-specifications":{
-   "component": "ProductSpecifications"
- }
+2. Add the `product-specifications` block to any block bellow `store.product` (Product template). For example:
+
+```json
+  "store.product": {
+    "children": [
+      "flex-layout.row#product",
+    ]
+  },
+  "flex-layout.row#product": {
+    "children": [
+      "product-specifications#product"
+    ]
+  },
+   "product-specifications#product": {
+    "props": {
+      "shoudCollapseOnTabChange": true,
+      "collapsible": "desktopOnly"
+    }
+  },
 ```
 
```

Comment:
I would add the following: 

⚠️ The block consumes data from the `product-context` therefore it should only be used in the `store.product` block, that is, in the product template.